### ISSUE TITLE
Data usage

### ISSUE THREAD

**Issue by thomasjobs** [mp-se/victron-receiver#5]
My Victron Receiver is using approx 350Mb during 24 hours. I'm using default settings for Push. To reduce amount of data sent should I adjust "Push timeout" or "Push minimum resend time"?

**Comment by mp-se** [comment]
Minimum resend time

**Comment by thomasjobs** [comment]
> Minimum resend time

Thanks, will test :-)

**Comment by mp-se** [comment]
Currently the max is 1800 seconds (30 min) but I can adjust that in the next version

**Comment by thomasjobs** [comment]
> Currently the max is 1800 seconds (30 min) but I can adjust that in the next version

I will try different settings......as the device is installed in my motorhome update is not critical. I will try with 1 min, 5min and 30 min. I report report here data usage.

**Comment by mp-se** [comment]
Its probably possible to add compression and reduce the data transmitted, json/text can be quite large.

**Comment by mp-se** [comment]
I look into adding compression and that would not really make any difference, the data payload is so small since every sensor needs to have its own request. The options are to limit what sensor values are sent or do less frequent updates.

**Comment by thomasjobs** [comment]
Have checked the data consumption. With ”Minimum resend time” at 30sec - device is using 0.6-0.7Gb during 24hrs. Whit 600sec device is using 30Mb.

**Comment by mp-se** [comment]
The rest will use more data than mqtt, the only other option is to reduce what sensor values to send

**Comment by mp-se** [comment]
Is this something that needs to be handled ?

**Comment by thomasjobs** [comment]
> Is this something that needs to be handled ?

No, reporting every 10-30min will give acceptable readings and data usage.
